includes/admin.php

```diff
@@ -16,8 +16,11 @@
  */
 function setup() {
 	add_action( 'add_meta_boxes', __NAMESPACE__ . '\\add_meta_box' );
-	add_action( 'save_post', __NAMESPACE__ . '\\handle_redirect_saving', 13 );
+	add_action( 'admin_enqueue_scripts', __NAMESPACE__ . '\\enqueue_scripts' );
+	add_action( 'manage_' . Redirects_Post_Type\SLUG . '_posts_custom_column', __NAMESPACE__ . '\\posts_columns_content', 10, 2 );
+	add_filter( 'manage_' . Redirects_Post_Type\SLUG . '_posts_columns', __NAMESPACE__ . '\\filter_posts_columns' );
 	add_action( 'plugins_loaded', __NAMESPACE__ . '\\load_plugin_textdomain' );
+	add_action( 'save_post', __NAMESPACE__ . '\\handle_redirect_saving', 13 );
```

Comment:
Why is this specific priority chosen over the default one? A comment would be appropriate.

Reply:
It's marked as changed because I put them in alphabetical order - the priority was already set to that, not sure why either.